### ISSUE TITLE
[Bug]: Should Radio button fix has been released?

### ISSUE THREAD

**Issue by BenceLovas** [carbon-design-system/carbon#15992]
### Question for Carbon

Hi,
I've opened an issue about the radio button component. https://github.com/carbon-design-system/carbon/issues/15894
It has been fixed, and based on the commit history, it should have been released in 1.53.0, but neither I can see it in the release notes, nor it works when I update the library. I might guessed it wrong by assuming that every commit gets released.
So, has it been released? 

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)

**Comment by tay1orjones** [comment]
@BenceLovas Based on the info on the [page for the merge commit](https://github.com/carbon-design-system/carbon/commit/7a9e2e645be3616108351f863d31e91122cb6844), that commit was first made available in v1.53.0-rc.0. This means it was also included in everything after: 1.53.0, and the patch we just cut, 1.53.1.

![image](https://github.com/carbon-design-system/carbon/assets/3360588/f238c918-95c4-4bcd-b6fe-afd8b4a14631)


Viewing [the published package on unpkg](https://unpkg.com/browse/@carbon/styles@1.53.0/scss/components/checkbox/_checkbox.scss) I can see that it's there (search for `print-color-adjust`)

Could you reproduce the issue in a [stackblitz](https://stackblitz.com/github/carbon-design-system/sandboxes/tree/main/react/latest/react-18-vite/?file=src%2Findex.scss,src%2FApp.jsx&preset=node=) showing how it's not working, and post the link back here? It's hard to know exactly what's going wrong without seeing the issue firsthand.

**Comment by BenceLovas** [comment]
@tay1orjones My original issue was about the Radio Button component, but it also has the `print-color-adjust` added to the [scss file](https://unpkg.com/browse/@carbon/styles@1.53.0/scss/components/radio-button/_radio-button.scss). 

The issue was (and still is) that the selected radio button has not been shown on the print preview (<kbd>⌘ Command</kbd> + <kbd>P</kbd>), hence the saved pdf won't have it either.

Reproduction: https://codesandbox.io/p/devbox/goofy-liskov-2ljg99?file=%2Fsrc%2FExample%2FExample.jsx%3A11%2C19
If you open print preview (it's a bit cluttered in this case), you can see that the radio button selection is not reflected (all circles are empty).

**Comment by tay1orjones** [comment]
It looks like that's isolated to the codesandbox environment. They're showing in the [story](https://react.carbondesignsystem.com/iframe.html?args=&id=components-radiobutton--default&viewMode=story)

<img width="1302" alt="image" src="https://github.com/carbon-design-system/carbon/assets/3360588/f95ac5f9-eb1a-480b-b7fd-58558ab1d58b">

**Comment by BenceLovas** [comment]
@tay1orjones It's weird, you're right it's showing up on your Storybook, but I tried it on Stackblitz as well and it was not woking: https://stackblitz.com/edit/vitejs-vite-6byqqw?file=src%2FApp.scss,src%2FApp.jsx&terminal=dev
I also tried it in two separate React projects and it wasn't working either.
Can it be that your Storybook example is somehow differs from what I was reproducing?

**Comment by guidari** [comment]
Yeah. Looks like it is only working on Storybook for some reason. 🤔
